client/app/widgets/Resources/index.jsx

```diff
@@ -38,8 +40,21 @@ export class Resources extends React.Component<Props, State> {
     this.state = { checkboxes: this.createCheckboxes() };
   }
 
+  componentDidUpdate() {
```

Comment:
Are you planning on adding tests for this in another PR? We should definitely be testing this!

Reply:
TBH, I didn't know how to test it :P
But I can give it a try in this same PR I will just need more time (probably this weekend) in order to do some research about how to test it.

Reply:
No worries! Let me know if you need any help, I'm happy to pair on it!

Reply:
Hi @julianguyen, I asked a frontend friend for some guidance and he ended up doing some pairing with me. Because of that, we wrote the specs and also changed a little bit so `history` could be "mockable".